### ISSUE TITLE
Couldn't find preset "babili" relative to directory

### ISSUE THREAD

**Issue by jdcrecur** [srod/node-minify#133]
I am using your node-minify package within my own npm package called quilk. It works flawlessly except on the babili compressor.

The idea with my package is you install it globally and it basically exposes other helpful modules, like yours.

The problem is, when I try to use the babili compressor i get an error:
```
Error: Couldn't find preset "babili" relative to directory "/mnt/4E6E45D36E45B48D/project"
```
If I install node-minify local to the current project it works perfectly. The odd thing is the uglify js compressor does not have the problem when used in quilk... any ideas?

This is the module in quilk that throws the error:  https://github.com/jdcrecur/quilk/blob/master/lib/modules/node_minify.js

**Comment by srod** [comment]
Do you have a sample code example to use quilk with node-minify and that throw that error ?
Thanks !

**Comment by jdcrecur** [comment]
Hi, yea sure. Here is an example quilk.js file. It will only run the one module, that being node_minify and should call the babili compressor without trying to use a .babelrc file, so presets need to be installed. 

It will look for a file in  /build/js/app.js  relative to your quilk.js file run it through your node-minify package and then it should write back to the same path.

```
module.exports = {
  modules : [
    {
      name: "minify the app js",
      module: "node_minify",
      type:"babili",
      input:  "/build/js/app.js",
      target: "/build/js/app.js"
    }
  ],

  dont_watch: [
    "/build/"
  ],

  developers : {
    default : {
      platform : "linux",
      notifier : {
        on  : false
      }
    }
  }
};
```

I have another few modules in quilk that use babelify directly with browserify and i don't have any issues.

It just seems to be this one compressor from within node-minify. I just republished quilk to not spew out so many errors on this compressor.

**Comment by srod** [comment]
Hi, sorry for delay.

For now I can see that babel is looking for preset babili relative to the directory where you are launching it.
So if you do a `npm install babel-preset-babili` in the folder, it's ok.

I'm going to search why babel can't find the preset directly in the quilk folder.

**Comment by srod** [comment]
Hi, I've published 2.0.4, can you try it ?

**Comment by jdcrecur** [comment]
Great, that has fixed things. Thanks a lot, I will publish quilk with the new version of node-minify.
